e2e-tests/common/BrowserWindow.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
```

Comment:
You don't need this anymore if it's a library.

Also, maybe let's call the file lower_case as it's more pythonic.